### PR TITLE
0.11 release blog

### PR DESCRIPTION
**Pull request by yuzisun**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/modelserving/v1beta1/llm/vllm/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Serving LLM models can be surprisingly slow even on high end GPUs, [vLLM](https://github.com/vllm-project/vllm) is a fast and easy-to-use LLM inference engine. It can achieve 10x-20x higher throughput than Huggingface transformers.
+It supports [continuous batching](https://www.anyscale.com/blog/continuous-batching-llm-inference) for increased throughput and GPU utilization,
+[paged attention](https://vllm.ai) to address the memory bottleneck where in the autoregressive decoding process all the attention key value tensors(KV Cache) are kept in the GPU memory to generate next tokens.
+
+You can deploy the LLaMA model with built vLLM inference server container image using the `InferenceService` yaml API spec. 
+We have work in progress integrating `vLLM` with `Open Inference Protocol` and KServe observability stack.
+
+The LLaMA model can be downloaded from [huggingface](https://huggingface.co/meta-llama/Llama-2-7b) and upload to your cloud storage.
+
+=== "Yaml"
```
Comment (rachitchauhan43) [inline docs/modelserving/v1beta1/llm/vllm/README.md:11]: YAML is not correctly rendering.

Screenshot 2023-10-24 at 10 10 46 AM

File: docs/modelserving/v1beta1/llm/vllm/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+## Deploy the LLaMA model with vLLM Runtime
+Serving LLM models can be surprisingly slow even on high end GPUs, [vLLM](https://github.com/vllm-project/vllm) is a fast and easy-to-use LLM inference engine. It can achieve 10x-20x higher throughput than Huggingface transformers.
+It supports [continuous batching](https://www.anyscale.com/blog/continuous-batching-llm-inference) for increased throughput and GPU utilization,
+[paged attention](https://vllm.ai) to address the memory bottleneck where in the autoregressive decoding process all the attention key value tensors(KV Cache) are kept in the GPU memory to generate next tokens.
+
+You can deploy the LLaMA model with built vLLM inference server container image using the `InferenceService` yaml API spec. 
+We have work in progress integrating `vLLM` with `Open Inference Protocol` and KServe observability stack.
+
+The LLaMA model can be downloaded from [huggingface](https://huggingface.co/meta-llama/Llama-2-7b) and upload to your cloud storage.
```
Comment (rachitchauhan43) [inline docs/modelserving/v1beta1/llm/vllm/README.md:9]:
```suggestion
The Llama-2-7b model can be downloaded from [huggingface](https://huggingface.co/meta-llama/Llama-2-7b) and upload to your cloud storage.
```

Just adding to explicitly call out that the benchmarking is for 7b model.

**Review by rachitchauhan43 — APPROVED** [review]
LGTM

File: docs/blog/articles/2023-10-08-KServe-0.11-release.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+# Announcing: KServe v0.11
+
+We are excited to announce the release of KServe 0.11, in this release we introduced LLM runtimes, made enhancements to the KServe control plane, Python SDK Open Inference Protocol support and dependency managemenet. 
+For ModelMesh we have added features PVC, HPA, payload logging to ensure feature parity with KServe.
+
+
+Here is a summary of the key changes:
+
+### KServe Core Inference Enhancements
```
Comment (ckadner) [inline docs/blog/articles/2023-10-08-KServe-0.11-release.md:9]: We are jumping from H1 (`#`) to H3 (`###`) level headings here.  Should we have `## KSreve Updates` above here like we have  `## ModelMesh Updates` below

File: docs/blog/articles/2023-10-08-KServe-0.11-release.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
+# Announcing: KServe v0.11
+
+We are excited to announce the release of KServe 0.11, in this release we introduced LLM runtimes, made enhancements to the KServe control plane, Python SDK Open Inference Protocol support and dependency managemenet.
```
Comment (ckadner) [inline docs/blog/articles/2023-10-08-KServe-0.11-release.md:3]: most people would probably know it already, but should we spell out "Large Language Model (LLM)" at least once?

File: docs/blog/articles/2023-10-08-KServe-0.11-release.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+A new [Grafana dashboard](https://github.com/kserve/modelmesh-serving/blob/main/docs/monitoring.md#import-the-grafana-dashboard) was added to display the comprehensive set of [Prometheus metrics](https://github.com/kserve/modelmesh-serving/blob/main/docs/monitoring.md) like model loading
+and unloading rates, internal queuing delays, capacity and usage, cache state, etc. to monitor the general health of the ModelMesh Serving deployment.
+
+The new [`PayloadProcessor` interface](https://github.com/kserve/modelmesh/blob/main/src/main/java/com/ibm/watson/modelmesh/payload/) can be implemented to log prediction requests and responses, to create data sinks for data visualization, for model quality assessment, or for drift and outlier detection by external monitoring systems.
+
+## :warning: What's Changed
```
Comment (ckadner) [inline docs/blog/articles/2023-10-08-KServe-0.11-release.md:118]: this looks a little off, could we remove the :warning: emoji, or move it to the end

File: docs/modelserving/v1beta1/llm/vllm/vllm.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+          limits:
+            cpu: "4"
+            memory: 50Gi
+            nvidia.com/gpu: "1"
+          requests:
+            cpu: "1"
+            memory: 50Gi
+            nvidia.com/gpu: "1"
+    maxReplicas: 1
+    minReplicas: 1
```
Comment (ckadner) [inline docs/modelserving/v1beta1/llm/vllm/vllm.yaml:32]: I think we need a newline at the end of file here, to allow concatenating manifest YAML files

**Review by ckadner — CHANGES_REQUESTED** [review]
looks good, just one little nit I failed to notice earlier

**Review by ckadner — APPROVED** [review]
/lgtm

**Comment by ckadner** [comment]
Looks good. Thanks Dan!

**Pull request merged**